### PR TITLE
removing ambiguity in readme

### PR DESCRIPTION
**Pull request by meteora2002**
this is my first github contribution ,so please feel free to give your constructive feedback

### CODE REVIEW

**Comment by tuladhar** [comment]
Congratulation @meteora2002 on your first contributions. I've left some comments, please check.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
-# Multi-threaded Google Ban Checker
+# Multi-threaded Google  Ban Checker
 
 
+# This is a Google account ban checker
```
Comment (tuladhar) [inline README.md]: I suggest, these to be added under NOTES. For e.g:

**NOTE:**
- This is not a Google account, domain or advertisements ban checker

File: README.md [hunk 2]
```diff
@@ -22,6 +27,9 @@ $ cat PROXIES.txt | python main.py --realtime
 ```
 
 #### `PROXIES.txt` should be in following format (one proxy per line):
+#### IMPORTANT NOTE
```
Comment (tuladhar) [inline README.md]: Similarly to above, let's use bolded NOTE section, and not make the text very big.

File: README.md [hunk 3]
```diff
@@ -31,8 +39,9 @@ username:password@proxy2:port
 #### Google Keywords and User Agents used by the script:
 - `helper/keywords.txt`
 - `helper/useragents.txt`
-
-Please update these files as needed.
+####PLease make sure your username and password exist here before using this
```
Comment (tuladhar) [inline README.md:39]: Similarly as above, let's add it to bolded NOTES section.

File: README.md [hunk 4]
```diff
@@ -62,6 +71,7 @@ How it works?
 
 **NOTE**:
 - If ban is detected at any stage no further requests will be made.
+- You will be notified once a ban is detected
```
Comment (tuladhar) [inline README.md:71]: This looks good.

File: README.md [hunk 5]
```diff
@@ -71,3 +81,6 @@ How it works?
 
 ## Contributors
 - See [Contributors.md](Contributors.md)
+
+## TIPS FOR UPDATING
```
Comment (tuladhar) [inline README.md]: Let's remove this, as it's not mandatory to mention it.

**Review by tuladhar — CHANGES_REQUESTED** [review]

**Comment by meteora2002** [comment]
done

**Comment by meteora2002** [comment]
@tuladhar please accept the pull requests

**Comment by meteora2002** [comment]
@tuladhar  please check my pull request now

**Comment by meteora2002** [comment]
@tuladhar  please check my pull request

File: README.md [hunk 6]
```diff
@@ -31,8 +36,9 @@ username:password@proxy2:port
 #### Google Keywords and User Agents used by the script:
 - `helper/keywords.txt`
 - `helper/useragents.txt`
-
-Please update these files as needed.
+####PLease make sure your username and password exist here before using this
+####Please update these files as needed.
+####Always update the files before using this
```
Comment (tuladhar) [inline README.md:41]: Let's also add this to note section.

**Review by tuladhar — CHANGES_REQUESTED** [review]
Left one comment. Besides that, looks good to merge!

**Comment by meteora2002** [comment]
done

**Comment by meteora2002** [comment]
@tuladhar please check it now

**Review by tuladhar — APPROVED** [review]

**Comment by tuladhar** [comment]
Thanks @meteora2002. Merged!

**Pull request merged**